api/src/test/java/com/epam/pipeline/manager/issue/IssueManagerTest.java

```diff
@@ -410,6 +417,22 @@ public void updateCommentWithAttachments() throws InterruptedException {
                                                                             Mockito.anyList(), Mockito.anyBoolean());
     }
 
+    @Test(expected = AccessDeniedException.class)
+    @Transactional(propagation = Propagation.REQUIRES_NEW, rollbackFor = Throwable.class)
+    @WithMockUser(username = TEST_USER)
+    public void testDeleteAttachmentFail() {
+        attachmentFileManager.deleteAttachment(testAttachment.getId());
+    }
+
+    @Test
+    @Transactional(propagation = Propagation.REQUIRES_NEW, rollbackFor = Throwable.class)
+    @WithMockUser(username = AUTHOR)
+    public void testDeleteAttachment() {
+        final AttachmentFileManager mockAttachmentFileManager = mock(AttachmentFileManager.class);
+        mockAttachmentFileManager.deleteAttachment(testAttachment.getId());
+        verify(mockAttachmentFileManager).deleteAttachment(testAttachment.getId());
+    }
+
```

Comment:
These tests basically don't check anything. It looks like the proper place for them is `AttachmentFileManagerTest`. But the tests should be rewritten without Spring being used. See other tests in `AttachmentFileManagerTest`.